### ISSUE TITLE
Screen-reader announces wrong name for first table cell

### ISSUE THREAD

**Issue by jelbourn** [angular/components#11136]
### Summary
Upon Screen Reader focus set on column 1, row 1 i.e. 'dessert (100g) ' , observe Screen Reader does not announce the name correctly. 

### Reproduction on material.angular.io
1. Linear navigate to ' Components ' menu item and click 'Enter' to open components page.                                                                                                                                                                                                                                          
2. Linear navigate forward and set focus on expand collapse left hand menu item ' Navigation'                                                                                                                                                                                                                                  
3. Click 'Enter' key to expand OR collapse this menu item                                                                                                                                                                                                                                                                                       
4. Linear navigate into 'sort header' & click ENTER keystroke.                                                                                                                                                                                                                                                                                   
5. Navigate into tabs "Examples" and Click ENTER                                                                                                                                                                                                                                                                                 
6. Navigate and set focus on column 1, row 1 i.e. 'dessert (100g) '

**Comment by jelbourn** [comment]
Tested this today, seems to no longer be an issue.
